mdpow/restart.py

```diff
@@ -179,6 +186,17 @@ def __init__(self, *args, **kwargs):
             self.journal = Journal(self.protocols)
         super(Journalled, self).__init__(*args, **kwargs)
 
+    @staticmethod
+    def get_pickle():
+        if sys.version_info.major == 3:
+            import _pickle as pickle
```

Comment:
Ugly compatibility code probably could use six

Reply:
use six if possible — `from six.moves import pickle` probably, see docs and/or try it out interactively